botogram/frozenbot.py

```diff
@@ -226,7 +226,11 @@ def _(self, message, **args):
 
     def available_commands(self, all=False):
         """Get a list of the commands this bot implements"""
-        for command in self._commands.values():
+        s = sorted({cmd for cmd in self._commands.values()},
```

Comment:
Here, you can directly use `sorted(self._commands.values(), key=lambda command: command.name)` instead of the set comprehension.